web/src/main/java/com/epam/esm/controller/CertificateRESTController.java

```diff
@@ -0,0 +1,139 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.entity.Certificate;
+import com.epam.esm.mapper.impl.certificateMapper.CreateCertificateFromOnlyCertificateMapper;
+import com.epam.esm.model.certificate.ModelCertificate;
+import com.epam.esm.model.certificate.OnlyCertificate;
+import com.epam.esm.pagination.Pagination;
+import com.epam.esm.servise.impl.CertificateServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.Link;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/v1.1")
+public class CertificateRESTController {
+
+    private final CertificateServiceImpl service;
+    private final CreateCertificateFromOnlyCertificateMapper certificateMapper;
+
+    /**
+     * created certificate
+     * enter the following fields: name,description,duration,price
+     * dates computed automatically
+     *
+     * @param certificate the certificate
+     * @return new certificate
+     */
+    @PostMapping("/certificates")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Certificate addCertificate(@RequestBody Certificate certificate) {
+        service.saveEntity(certificate);
+        return certificate;
+    }
+
+
+    /**
+     * @param page the page
+     * @param size the size
+     * @return list certificate
+     */
+    @GetMapping("/certificates")
```

Comment:
all your endpoints in controlles starts with /certificates. So, consider moving this part into @RequestMapping("/api/v1.1") annotation on class level

Reply:
mode